redash/handlers/dashboards.py

```diff
@@ -57,6 +57,7 @@ def post(self):
                                      user=self.current_user,
                                      is_draft=True,
                                      layout='[]')
+        dashboard.record_changes(changed_by=self.current_user)
```

Comment:
I didn't see any additional dashboard references. Is recording a change to a dashboard already supported?